nimble/host/audio/syscfg.yml

```diff
@@ -17,5 +17,9 @@
 #
 
 syscfg.defs:
+    BLE_AUDIO_MAX_CODEC_RECORDS:
+        description: >
+            Maximum number of registered audio codecs.
+        value: 0
```

Comment:
does it make sense to include audio package and have no audio codecs registered?

Reply:
I think you can have app withouth codecs registered - it's not mandatory to register codecs you will use. So you can for example include this package, create broadcast application (bare bones, just sending advertising chain and BISes), use some codec to encode audio and not register it. But you need to register so it'll be visible in PACS (and also include PACS itself).

I think I should add compiler guards so ble_audio_codec.c is compiled only when this is >0